### ISSUE TITLE
Broken hashtag search

### ISSUE THREAD

**Issue by vitorgrs** [mastodon/mastodon#21780]
### Steps to reproduce the problem

1. Search for #mastodon on bolha.us 
2. There will be 0 results. 

### Expected behaviour

Show the allowed hashtags 

### Actual behaviour

Show zero hashtags

### Detailed description

 At our instance on bolha.us, we have set to automatically approve hashtags only recently, and thus, these prior hashtags that was in use prior to that doesn't show on search.

We already allowed ALL of these old hashtags, and yet it doesn't show on search.   

![image](https://user-images.githubusercontent.com/9909446/204216257-eb9df889-1745-45d3-8210-a1f005da9ff1.png)

As you can see, it shows as set to show on search, but obviously is not.

![image](https://user-images.githubusercontent.com/9909446/204216331-b47a7c75-454c-4139-ada3-766f0a9b70a7.png)

The solution that I found is to manually enter on each hashtag, disable to show on search, save. And then allow to show on search, save. AND THEN it shows on search:

<img width="674" alt="image" src="https://user-images.githubusercontent.com/9909446/204217155-1a2dab91-5708-4ca7-8551-9dff9729f795.png">

So tl;dr new hashtags works fine, older one doesn't. How to set all the other ones to show on search? There's over 700 hashtags...

### Specifications

Mastodon 4.0.2
Windows 11, Edge 109.0.1495.2

**Comment by vitorgrs** [comment]
a `tootctl search deploy` fixed it.
